lib/schema.js

```diff
@@ -151,9 +152,28 @@ class Metaschema {
       if (checked.some(v => v === null)) return null;
       return checked;
     }
+    if (decorator === 'Enum') {
+      const { values } = def;
+      const valid = Enum.from(values).has(value);
+      if (!valid) return null;
+      return value;
+    }
+    if (decorator === 'Flags') {
+      const { values } = def;
+      let flags;
+      try {
+        flags = Flags.from(...values).from(...value);
```

Comment:
@o-rumiantsev I think it will be better to use `has`
```suggestion
        const flags = Flags.from(...values);
        if (value.some(v => !flags.has(v)) return null;
```
Though this will only work for array so perhaps you should use `iter`.

Reply:
`Flags.from`  returns a `FlagsClass` class, so in your suggestion variable `flags` wouldn't have method `has`

Reply:
I thought `has` is static, if it isn't I think we should make it so.